### DISCUSSION TITLE
Uavsar Interferograms

### DISCUSSION THREAD

**Discussion by ZachHoppinen** [isce-framework/isce2#472] [Q&A] [answered]
Uavsar is one of the sensors that the ReadMe lists as Isce being able to perform Insar on. 

> In its InSAR aspect ISCE supports data from ... ALOS, ALOS2, COSMO_SKYMED, ENVISAT, ERS, KOMPSAT5, RADARSAT1, RADARSAT2, RISAT1, Sentinel1, TERRASARX, **UAVSAR** and SAOCOM1A.

**If I have a stack of SLC images how do I generate an interferogram from them?** I have not been able to find any tutorials or examples of how to do this on either the isce2 or isce2-docs page. Can someone direct me at least to the right script or format for the XML file?

So far I have tried to use the one example XML file form the input_file directory and the isceApp.py to generate interferograms but I don't know if that is the correct script to be using or if it is which inputs to include. Here is my current XML file that I tried to adapt from the one example I could find:

<img width="281" alt="Screen Shot 2022-04-03 at 4 34 48 PM" src="https://user-images.githubusercontent.com/56941602/161451624-2592f957-dd23-4992-a86e-e681b161028f.png">
.

**Comment by EJFielding** [comment, marked as answer]
See https://github.com/isce-framework/isce2/discussions/469#discussioncomment-2502722
